feat : add login page

Kevin-Solomon opened the pull request:
# Login Page

![image](https://user-images.githubusercontent.com/80545228/152636065-e6162031-e66f-4c23-ad42-e36f605f5159.png)
[Login Page CodeSandbox Link](https://u4px6.csb.app/pages/login/login.html)

Kevin-Solomon committed:
feat : add login page

mohammed-sas reviewed (commented):
simple and clean ui.
suggestion : try different variant of white for background to improve contrast

SatwikaKacham commented on pages/login/login.css:
```diff
@@ -0,0 +1,73 @@
+.navbar .navbar-btn {
+  display: none;
+}
+@media (max-width: 768px) {
```
it would be better if u write media queries at the end of the CSS file. because nothing can override it.

Kevin-Solomon replied:
Done

SatwikaKacham reviewed (approved):
clean and neat work. good to merge

Kevin-Solomon committed:
style : changed background color of card

changed background color of the card to make it distinguishable. Also changed minimum width of the card
